dynatademand/api.py

```diff
@@ -158,6 +158,9 @@ def get_countries(self, **kwargs):
         )
         return self._api_get('/countries', kwargs)
 
+    def get_study_metadata(self):
+        return self._api_get('/studyMetadata')
```

Comment:
Is this metadata for a single project or all projects?

Reply:
It is for all the projects
https://developers.dynata.com/demand-api-reference/data_endpoints/metadata/study-metadata

Reply:
Can we add a test for this function? I know it's a simple API call, but we have tests for the other simple API calls as well for coverage.

Reply:
@DynataRidley, added test case for the studyMetaData endpoint.